test/legacy/roles/test_digital_ocean/tasks/main.yml

```diff
@@ -2,3 +2,5 @@
   include: tasks/sshkey.yml
```

Comment:
@BondAnthony @kbreit Once we get through the current PRs we should move all the DigtialOcean items from `test/legacy` to `test/integration/targets`

Reply:
I already have a PR to enable the new test method so once that’s merged it should be easy to integrate into this.